packages/maskbook/src/plugins/ITO/assets/usdcIcon.tsx

```diff
@@ -1,6 +1,8 @@
-export const USDCIcon = ({ size = 32 }: { size?: number }) => (
-    <>
-        <svg width={size} height={size} viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+import { SvgIcon, SvgIconProps } from '@material-ui/core'
+
+export const USDCIcon: React.FC = (props: SvgIconProps) => (
+    <SvgIcon {...props}>
+        <svg viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
             <g clip-path="url(#clip0)">
```

Comment:
id may conflict

provide an `lodash.uniqueId` on react hook?

see https://lodash.com/docs/4.17.15#uniqueId

CC @Jack-Works

Reply:
i've added ID

Reply:
if this ID is limited in this svg element, then it is ok